### DISCUSSION TITLE
通用设置里面定义的主题颜色，能应用到播放器按钮/音量控制条吗？

### DISCUSSION THREAD

**Discussion by doio** [the1812/Bilibili-Evolved#3180] [功能建议 / Ideas]
通用设置 — 主题颜色 设置了 #F06291，不过播放器的音量控制和按钮都还是默认的蓝色，不知道能不能也应用到这些控件上？
![Theme-Color](https://user-images.githubusercontent.com/4461202/160054717-724f67be-bd2e-48e1-b05b-09898ea380c2.png)
![Player-Buttons](https://user-images.githubusercontent.com/4461202/160054724-d39a3794-4aac-4e88-b0bd-94061a86dede.png)
![Player-Volume](https://user-images.githubusercontent.com/4461202/160054727-b613dfad-2826-489e-93d8-97d402b60764.png)
![Misc](https://user-images.githubusercontent.com/4461202/160056315-16ea794d-0f79-46db-8f19-4e4ec612a2db.png)
![Resolution](https://user-images.githubusercontent.com/4461202/160057083-79a00132-b3a9-49aa-9e7f-9b180bd206d7.png)

**Comment by Geeyun-JY3** [comment]
属于是夜间模式没适配了。

**Reply by the1812** [reply]
应该就是新版播放器

**Reply by Geeyun-JY3** [reply]
不是吧，界面大体都没问题。
我记得番剧这里没适配主题色有好一段时间了。

**Reply by the1812** [reply]
原来是番剧啊, 我以为是因为最近那个 3.x 视频区播放器

**Comment by doio** [comment]
啊，没留意到普通视频是没问题的，看来又是番剧的播放器是新版的缘故  @_@
